src/main/resources/config/application.yml

```diff
@@ -10,7 +10,6 @@ security:
       tokenInfoUri: ${TOKENINFO_URL}
 
 management:
-  port: 7979
```

Comment:
It's default is 'server.port'. So we have to configure it. Especially when 'security=false'